updater/lib/dependabot/sentry.rb

```diff
@@ -1,23 +1,33 @@
-# typed: false
+# typed: strong
 # frozen_string_literal: true
 
 require "raven"
+require "sorbet-runtime"
 
 # ExceptionSanitizer filters potential secrets/PII from exception payloads
 class ExceptionSanitizer < Raven::Processor
+  extend T::Sig
+
   REPO = %r{[\w.\-]+/([\w.\-]+)}
-  PATTERNS = {
-    auth_token: /(?:authorization|bearer):? (\w+)/i,
-    repo: %r{https://api\.github\.com/repos/#{REPO}|https://github\.com/#{REPO}}
-  }.freeze
+  PATTERNS = T.let(
+    {
+      auth_token: /(?:authorization|bearer):? (\w+)/i,
+      repo: %r{https://api\.github\.com/repos/#{REPO}|https://github\.com/#{REPO}}
+    }.freeze,
+    T::Hash[Symbol, Regexp]
+  )
 
+  sig do
+    params(data: T::Hash[Symbol, T.nilable(T::Hash[Symbol, T::Array[T::Hash[Symbol, String]]])])
+      .returns(T::Hash[Symbol, T.untyped])
+  end
   def process(data)
-    return data unless data[:exception] && data[:exception][:values]
+    return data unless data[:exception]&.[](:values)
```

Comment:
First time I've seen this syntax, might be more clear with `dig`.

```suggestion
    return data unless data.dig(:exception, :values)
```